### ISSUE TITLE
Build WG Meeting 2016-09-20

### ISSUE THREAD

**Issue by jbergstroem** [nodejs/build#498]
(better late than never)

**Meeting Time:** 8PM Tuesday September 20st UTC, or in your local timezone:
http://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Build+WG+Meeting&iso=20160920T20&p1=%3A

### Where
* Hangouts on Air, for participants: https://hangouts.google.com/call/hkmga2nqwbbllb72sgx4535fhye
* Youtube stream, for viewers:  https://www.youtube.com/watch?v=qFJh5Wagdhs
* Minutes document: https://docs.google.com/document/d/1mz-gvvFlaZaz1Fdl_fM5oLGtOHMRugul4XjkR537UnM/edit?usp=sharing

### Agenda
Extracted from `wg-agenda` [issues](https://github.com/nodejs/build/issues?q=is%3Aopen+is%3Aissue+label%3Awg-agenda) and [pull requests](https://github.com/nodejs/build/pulls?q=is%3Aopen+label%3Awg-agenda+is%3Apr) from this repo.

- Invite @Trott to the build team #497
- build group membership reminder! #489
- Create proposal for list of officially supported platforms #488 
- TAP Plugin issues on Jenkins #453
- How can a project qualify for access to the build infrastructure? #448
- rsync endpoint to mirror the releases enhancement infra #55

If there are any items that you would like to discuss at the meeting, please tag new or existing issues with the `wg-agenda` label, or simply post them in a comment here.

All welcome of course, primarily @nodejs/build focused but we're happy to expand.

**Comment by TheAlphaNerd** [comment]
I'm going to be a bit late to today's meeting, if I'm able to make it at all. Sorry for the last minute notice.

Re: tap plugin

TLDR; we need to update the node test runner to report tap using YAMLish for the stack traces. Once that is done the tap -> junit converter should work using an off the shelf module.

I'm also curious where we are on better OSX ci stuff. We've gotten burnt by quite a bit of stuff on more modern OSX than we are currently testing.

**Comment by rvagg** [comment]
live @ https://www.youtube.com/watch?v=qFJh5Wagdhs (link is wrong in OP)

**Comment by mhdawson** [comment]
https://github.com/nodejs/build/pull/502
